CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
+  * You can now enable [Federal Information Process Standard (FIPS)](https://aka.ms/aks/enable-fips) when using [Arm64 VM SKUs](https://aka.ms/aks/arm64) in Azure Linux 3.0 node pools in Kubernetes version 1.31+.
+  * Enable Kata Confidential mounts for Azure File CSI driver on AKS 1.32
+
+* Preview Features:
+
+* Bug Fixes:
+  * Cost Analysis add-on updated to v0.0.22 to fix [CVE-2025-22866](https://pkg.go.dev/vuln/GO-2025-3447)
+  * IP masq agent updated to 0.1.15-2 to fix [CVE-2024-45338](https://nvd.nist.gov/vuln/detail/CVE-2024-45338)
+  * Application routing updated to v0.2.1-patch-8 for Kubernetes below 1.30 and to v0.2.3-patch-6 for Kubernetes 1.30+. This updates ingress-nginx to v1.11.5 to fix [CVE-2025-1097](https://nvd.nist.gov/vuln/detail/CVE-2025-1097), [CVE-2025-1098](https://nvd.nist.gov/vuln/detail/CVE-2025-1098), [CVE-2025-1974](https://nvd.nist.gov/vuln/detail/CVE-2025-1974), [CVE-2025-24513](https://nvd.nist.gov/vuln/detail/CVE-2025-24513), and [CVE-2025-24514](https://nvd.nist.gov/vuln/detail/CVE-2025-24514).
+
+* Behavior Changes:
+  * Add node anti-affinity for FIPS-compliant nodes to prevent scheduling of retina-agent pods to stop CrashLoopBackOff on FIPS-enabled nodes whilst fix for Retina + FIPS is being rolled out.
+  * Updated Kubernetes Scheduler Topology Constraints default of maxSkew from 5 to 1 for zones in all AKS clusters.
+  * Increased tofqdns-endpoint-max-ip-per-hostname from 50 to 1000 and tofqdns-min-ttl from 0 to 3600 in Azure Cilium for better handling of large DNS responses and reduce DNS query load.
+  * Konnectivity agent will now scale based on cluster node count
+
+* Component Updates:
+  * Updated NPM to v1.5.45 to resolve [CVE-2025-22870](https://nvd.nist.gov/vuln/detail/CVE-2025-22870)
+  * Cilium updated to v1.17 so that L7 policy (http, kafka etc) can now be applied to a cluster when advancedNetworkPolicies is set.
+  * Windows GPU Device plugin updated to 0.0.17 to resolve [CVE-2025-22870](https://nvd.nist.gov/vuln/detail/CVE-2025-22870).
+  * Egress gateway updated to [0.019](https://github.com/Azure/kube-egress-gateway/releases/tag/v0.0.19)
+  * Eraser updated to v1.4.0-2 for Image Cleaner
+  * Retina updated to [v0.0.29](https://github.com/microsoft/retina/releases/tag/v0.0.29) on Linux and Windows.
+  * Cluster Autoscaler [updated](https://github.com/kubernetes/autoscaler/releases) to 1.29.5, 1.30.3, 1.31.1.
+  * Istio has been patched to 1.23.5 and 1.24.3
+  * Azure File & Disk CSI driver updated to v1.29.14, v1.30.10,  v1.31.6 & v1.32.1
+  * Azure Blob CSI driver apdated to v1.25.5 on AKS 1.31 & v1.26.2 on AKS 1.32
+  * AKS Azure Linux v2 image has been updated to [202504.06.0](vhd-notes/AzureLinux/202504.06.0.txt).
```

Comment:
Lets remove all the VHD / Agent baker announcements as they are not going to be up to date . Lets pls point customers to go real time with Release tracker : https://releases.aks.azure.com/webpage/index.html